app/lib/src/lib.rs

```diff
@@ -0,0 +1,28 @@
+// in Wii mode, enable `no_std` and import ogc_rs replacements of common functions.
+#![cfg_attr(feature = "wii", no_std)]
+#[cfg(feature = "wii")]
+pub use ogc_rs::{print, println};
+
+#[cfg(not(feature = "wii"))]
+pub use std::{print, println};
+
+
```

Comment:
Redundant whiteline ;)

Reply:
I've added #27 so we don't have to bicker too long about this kind of stuff 😛

Reply:
I had left this newline intentionally as separation between the declarations and the first functions, but I'll remove it 👌